AdobeStockStub/Model/FileGenerator.php

```diff
@@ -112,22 +82,16 @@ private function constructStubFileData(int $iterator): array
             ],
             'media_type_id' => 1,
             'content_type' => 'image/png',
-            'details_url' => $imageUrl,
+            'details_url' => $this->getImageUrl(),
             'premium_level_id' => 0,
         ];
     }
 
     /**
-     * Set stub images.
+     * @return string
      */
-    private function setStubImages()
+    private function getImageUrl(): string
     {
-        if (empty($this->stubImages)) {
-            $this->stubImages = [
-              $this->assetRepository->getUrl('Magento_AdobeStockStub::images/1.png'),
-              $this->assetRepository->getUrl('Magento_AdobeStockStub::images/2.png'),
-              $this->assetRepository->getUrl('Magento_AdobeStockStub::images/3.png')
-            ];
-        }
+        return $this->assetRepository->getUrl('Magento_AdobeStockStub::images/' . rand(1, 3) . '.png');
```

Comment:
Working on test coverage with the stub module I found that the number of images should be > than 3 :) Because one of the tests sorts images and asserts that the image # 2 in ASC sort will not be the same when DESC is applied.
ASC            DESC
1,2,3           3,2,1
So I introduced next: https://github.com/magento/adobe-stock-integration/pull/1545/files#diff-16e3c853263dc10709168cd7b54e87baR118

Reply:
@sivaschenko but this is mot critical in this PR. If it solves the main problem I will adjust this part with my changes in another PR.